pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>com.youengineering</groupId>
     <artifactId>openapi-maven-plugin</artifactId>
-    <version>0.0.11</version>
+    <version>0.1.0</version>
```

Comment:
If you increase the version here, you should (but don't have to) increase it in
- src/test/resources/jaxrs/pom.xml
- src/test/resources/spring/pom.xml
as well, just for clarity and consistency.